### DISCUSSION TITLE
Add Search and/or Filters to /admin/advanced/response_templates

### DISCUSSION THREAD

**Discussion by michael-tharrington** [forem/forem#16656] [Feature Requests]
We've recently updated our embed functionality and I need to update the moderator response template for "📹 Suggest to embed a video 📹" so that the syntax is up-to-date. However, I can't find this template in `/admin/advanced/response_templates` and there is no search or filtering. 

In order to find the template, I need to start clicking through pages and using `command + f` to see if there is a template with this title on the page. That's a time sink and not a great experience for Forem admin.

Below is a picture of the `/admin/advanced/response_templates` on our test Forem. There are only a handful of response templates on this Forem, so it's pretty easy to navigate, but you can imagine how difficult this page is to navigate when you have several pages of these templates to scroll through.

![Screen Shot 2022-02-21 at 11 56 19 AM](https://user-images.githubusercontent.com/16007075/154998588-6a2b4457-ab6e-427a-be3f-489bc6fa3935.png)

**Comment by ellativity** [comment]
If you search for "filter" in this repo's Discussions, it feels like filtering is something that would be beneficial across the admin and user-facing site.

**Reply by michael-tharrington** [reply]
You're right about that! I want mo filters.

**Reply by andypiper** [reply]
In one way, it's a good problem to have - lots of folks creating content in various areas of the site! Having better ways to "slice and dice" to find what you're looking for is the next step forward.
